src/Documentation/UserManual/DeviceCapistrano.dox

```diff
@@ -10,11 +10,14 @@
 \section CapistranoSupportedPlatforms Supported platforms
 
 - \ref PackageWin32
+- \ref PackageWin64
 
 \section CapistranoInstallation Installation
 
-- Requires SDK provided by Capistrano Labs for compilation and USB drivers
-  provided by Capistrano Labs for running (otherwise Plus application will not start due to missing WDAPI1010.dll).
+- Requires SDK provided by Capistrano Labs for compilation. Windows 64 bits platform only supported by cSDK2018.
+- Requires USB drivers provided by Capistrano Labs for running. There are specific USB drivers for each cSDK version.
+  - If using cSDK2018 or cSDK2016, Plus application will not start due to missing WDAPI1140.dll

```

Comment:
I’ll have to look again at the top of their files regarding the license and then potentially ask Capistrano directly. By the way, what’s the difference between PlusLib/Tools and the PerkLab PLTools repo? It looks like the Interson old SDK (iSDK) is within PLTools even though it’s Plus documentation page says ”Requires SDK provided by Interson”. Interson is a different company from Capistrano, but you’ll notice similarities between these SDKs since it originated from the same developer.

Reply:
@jamesobutler PLTools is the PerkLab's private repo for SDK's that have been authorized to them. PlusLib/Tools is for SDKs that are freely distributed.